web/src/app.scss

```diff
@@ -187,3 +187,31 @@ p {
   vertical-align: middle;
   color: branding.$eos-bc-gray-900;
 }
+
+ul.connections-datalist {
+  border: 0;
+  li {
+    border: 0;
+    // TODO: get rid of this nested overrides once the network overview UI is better defined
+    .pf-c-data-list__item-row {
+      padding-inline: 0;
+      .pf-c-data-list__item-content {
+        padding-block-end: 0;
+      }
+    }
+  }
+}
+
+.auto-modal-popup {
+  block-size: auto;
+}
+
+.medium-modal-popup {
+  min-block-size: 55vh;
+  max-block-size: 55vh;
+}
+
+.large-modal-popup {
+  min-block-size: 85vh;
+  max-block-size: 85vh;
```

Comment:
These height sizes were added to have _height-fixed_ popups, which is useful to avoid having a popup that changes its height while adding or removing dynamic fields, as happens now with the Connection Settings.

If you think other sizes are more appropriated, please speak up :wink: